----------------------------------------------------------------------
Revision: 2ed09e2720ad78b81a36d3b3d90602e8459b58b8
Parent:   29828d7d187ae07d441e863d23e631c24b7e1fea
Author:   yan12125@gmail.com
Date:     西元2019年06月13日 21時19分13秒
Branch:   net.venge.monotone

Changelog: 

Fix building with Apple Clang

Apparently Apple Clang supports __float128 now, and declarations like
`struct __float128` lead to compiler errors.

Changes against parent 29828d7d187ae07d441e863d23e631c24b7e1fea

  patched  src/base.hh
  patched  src/netxx/compat.h

============================================================
--- src/base.hh	2c6f22801aef99bb44b173fba7e4564ae4d9b95f
+++ src/base.hh	7731f4cec3edeed490b45f21009bf952402dd1ee
@@ -39,11 +39,6 @@
 #undef __STRICT_ANSI__
 #endif
 
-// Clang doesn't support __float128, but glibc needs it.
-#ifdef __clang__
-struct __float128;
-#endif
-
 #include <iosfwd>
 #include <string>  // it would be nice if there were a <stringfwd>
 
============================================================
--- src/netxx/compat.h	a7dc9d82de9bfc739303bb2730f74c68ecbfbbca
+++ src/netxx/compat.h	de965006ea6b46afb54802a44b93a56d8f23a86f
@@ -78,9 +78,4 @@ namespace std {
 
 #endif // __cplusplus
 
-// Clang doesn't support __float128, but glibc needs it.
-#ifdef __clang__
-struct __float128;
-#endif
-
 #endif // _compat_h__compatability_header
